fastlane/framework.rb

```diff
@@ -92,6 +92,8 @@ def self.all
 
         frameworks["WireAPI"].add_dependency(frameworks["WireFoundation"])
 
+        frameworks["WireAnalytics"].add_dependency(frameworks["WireFoundation"])
```

Comment:
I believe this is:
```suggestion
        frameworks["WireAnalytics"].add_dependency(frameworks["WireLogging"])
```
now that WireLogging is it's own package

Reply:
fixed, thanks.